pkg/ostree/client.js

```diff
@@ -448,13 +457,13 @@ define([
             local_running = method + ":" + os;
             var transaction_client = null;
             var subscription = null;
-            var dp = $.Deferred();
+            var dp = cockpit.defer();
             var i;
             var reboot = false;
 
-            if ($.isArray(method_args)) {
+            if (Array.isArray(method_args)) {
                 for (i = 0; i < method_args.length; i++) {
-                    if ($.isPlainObject(method_args[i])) {
+                    if ("reboot" in method_args[i]) {
```

Comment:
This isn't quite the same thing and causes and error when method_args[i] is a string.